src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/tests/latest/test_dls_commands.py

```diff
@@ -375,16 +375,14 @@ def test_dls_account_mgmt(self, resource_group):
             'subnet_id': '/subscriptions/9e1f0ab2-2f85-49de-9677-9da6f829b914/resourceGroups/lewu-rg/providers/Microsoft.Network/virtualNetworks/lewuVNET/subnets/default',
             'updated_subnet_id': '/subscriptions/9e1f0ab2-2f85-49de-9677-9da6f829b914/resourceGroups/lewu-rg/providers/Microsoft.Network/virtualNetworks/lewuVNET/subnets/updatedSubnetId'
         })
-        self.cmd('dls account network-rule create -g {rg} -n {dls} --vnet-name {vnet} --subnet {subnet_id}')
-        self.cmd('dls account network-rule show -g {rg} -n {dls} --vnet-name {vnet}', checks=[
+        self.cmd('dls account network-rule create -g {rg} -n {dls} --virtual-network-rule-name {vnet} --subnet {subnet_id}')
+        self.cmd('dls account network-rule show -g {rg} -n {dls} --virtual-network-rule-name {vnet}', checks=[
```

Comment:
Here, `virtual_network_rule_name` should be aliased to `--name/-n` not DLS.

Reply:
Thanks. should I change from ```--virtual-network-rule-name``` to ```--name``` ?  What do you mean by ```not DLS```

Reply:
Your example shows `-n {dls}` but name should refer to the rule name in this case, not the DLS account instance.